reader/views.py

```diff
@@ -50,7 +51,8 @@ def chapter_page(request, slug, vol, num, page):
     chapters['prev'] = prev_in_order(chapters['curr'], qs=chapters['all'])
     all_pages = chapters['curr'].pages.all()
     if page > len(all_pages):
-        raise Http404
+        return render(request, 'series.html', {'series': _series, 'marked': marked})
```

Comment:
This isn't ideal because if a user visits the page explicitly, it should raise 404. The only situation in which they should be sent to the series is when they're navigating through the reader and hit the end of the chapter.

Reply:
Oh, okay , ill try to find a solution to this. Sorry, i'm a bit new to web development and Django in general